examples/getting_started/plot_quick_start.py

```diff
@@ -0,0 +1,62 @@
+"""
+.. _example_quick_start:
+
+===========
+Quick start
+===========
+"""
+
+# %%
+# From your Python code, create and load a skore :class:`~skore.Project`:
+
+# %%
+import skore
+
+my_project = skore.create("quick_start", overwrite=True)
+
+# %%
+# This will create a skore project directory named ``quick_start.skore`` in your
+# current working directory.
+
+# %%
+# Evaluate your model using skore's :class:`~skore.CrossValidationReporter`:
+
+# %%
+from sklearn.datasets import load_iris
+from sklearn.svm import SVC
+
+X, y = load_iris(return_X_y=True)
+clf = SVC(kernel="linear", C=1, random_state=0)
```

Comment:
Do not use an SVC but rather a LogisticRegression

Reply:
With the following code:
```python
from sklearn.datasets import load_iris
from sklearn.linear_model import LogisticRegression

X, y = load_iris(return_X_y=True)

clf = LogisticRegression()

reporter = skore.CrossValidationReporter(clf, X, y, cv=5)
```

I get the following error:
```
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:469: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
```

Reply:
Yes it is bug introduce some times ago. I open a PR to fix it here: https://github.com/probabl-ai/skore/pull/1009

Reply:
Created a separate issue: https://github.com/probabl-ai/skore/issues/1018